diff -uNr openswan-2.2.0.orig/programs/auto/auto.in openswan-2.2.0/programs/auto/auto.in
--- openswan-2.2.0.orig/programs/auto/auto.in	2004-06-27 22:35:00.000000000 +0200
+++ openswan-2.2.0/programs/auto/auto.in	2004-11-24 20:32:37.450107496 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # user interface to automatic keying and Pluto in general
 # Copyright (C) 1998, 1999, 2000  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/barf/barf.in openswan-2.2.0/programs/barf/barf.in
--- openswan-2.2.0.orig/programs/barf/barf.in	2004-06-08 03:02:11.000000000 +0200
+++ openswan-2.2.0/programs/barf/barf.in	2004-11-24 20:32:52.928754384 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # dump assorted information of use in debugging
 # Copyright (C) 1998, 1999  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_confread/_confread.in openswan-2.2.0/programs/_confread/_confread.in
--- openswan-2.2.0.orig/programs/_confread/_confread.in	2004-06-27 22:32:53.000000000 +0200
+++ openswan-2.2.0/programs/_confread/_confread.in	2004-11-24 20:33:36.302160624 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # configuration-file reader utility
 # Copyright (C) 1999-2002  Henry Spencer.
 #
diff -uNr openswan-2.2.0.orig/programs/_confread/randomize openswan-2.2.0/programs/_confread/randomize
--- openswan-2.2.0.orig/programs/_confread/randomize	2002-04-24 09:36:05.000000000 +0200
+++ openswan-2.2.0/programs/_confread/randomize	2004-11-24 20:33:44.146968032 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # internal utility for putting random keys into sample configuration file
 # Copyright (C) 1998, 1999  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_include/_include.in openswan-2.2.0/programs/_include/_include.in
--- openswan-2.2.0.orig/programs/_include/_include.in	2003-01-06 22:44:04.000000000 +0100
+++ openswan-2.2.0/programs/_include/_include.in	2004-11-24 20:33:53.632526008 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # implements nested file inclusion for control files, including wildcarding
 # Copyright (C) 1998, 1999  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/ipsec/ipsec.in openswan-2.2.0/programs/ipsec/ipsec.in
--- openswan-2.2.0.orig/programs/ipsec/ipsec.in	2004-08-22 17:41:28.000000000 +0200
+++ openswan-2.2.0/programs/ipsec/ipsec.in	2004-11-24 20:35:35.581027480 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # prefix command to run stuff from our programs directory
 # Copyright (C) 1998-2002  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_keycensor/_keycensor.in openswan-2.2.0/programs/_keycensor/_keycensor.in
--- openswan-2.2.0.orig/programs/_keycensor/_keycensor.in	2002-04-24 09:36:05.000000000 +0200
+++ openswan-2.2.0/programs/_keycensor/_keycensor.in	2004-11-24 20:34:00.305511560 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # implements key censoring for barf
 # Copyright (C) 1999, 2002  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/livetest/livetest.in openswan-2.2.0/programs/livetest/livetest.in
--- openswan-2.2.0.orig/programs/livetest/livetest.in	2004-07-15 05:54:13.000000000 +0200
+++ openswan-2.2.0/programs/livetest/livetest.in	2004-11-24 20:35:50.214802808 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 # Copyright (C) 2004  Michael Richardson	<mcr@xelerance.com>
 # 
diff -uNr openswan-2.2.0.orig/programs/look/look.in openswan-2.2.0/programs/look/look.in
--- openswan-2.2.0.orig/programs/look/look.in	2003-10-31 03:32:42.000000000 +0100
+++ openswan-2.2.0/programs/look/look.in	2004-11-24 20:37:06.044274976 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # quick look at current connections and related information
 # Copyright (C) 1998, 1999  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/mailkey/mailkey.in openswan-2.2.0/programs/mailkey/mailkey.in
--- openswan-2.2.0.orig/programs/mailkey/mailkey.in	2003-06-30 07:34:22.000000000 +0200
+++ openswan-2.2.0/programs/mailkey/mailkey.in	2004-11-24 20:36:38.811415000 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # mail OE DNS RR info to relevent administrator
 #
 # Copyright (C) 2003 Sam Sgro <sam@freeswan.org>
@@ -203,7 +203,7 @@
 # -s - Subject line. By extending it, we can "hack" the mail program to
 # include a customized Reply-To header.
 
-mailbody="#!/bin/sh
+mailbody="#!/bin/bash
 #
 # Edit this variable to send this message to an alternate destination
 contact_email=$proposed_email
diff -uNr openswan-2.2.0.orig/programs/manual/manual.in openswan-2.2.0/programs/manual/manual.in
--- openswan-2.2.0.orig/programs/manual/manual.in	2002-10-22 11:07:27.000000000 +0200
+++ openswan-2.2.0/programs/manual/manual.in	2004-11-24 20:37:19.621210968 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # user interface to manual keying
 # Copyright (C) 1998, 1999  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/newhostkey/newhostkey.in openswan-2.2.0/programs/newhostkey/newhostkey.in
--- openswan-2.2.0.orig/programs/newhostkey/newhostkey.in	2002-09-14 00:27:31.000000000 +0200
+++ openswan-2.2.0/programs/newhostkey/newhostkey.in	2004-11-24 20:37:48.029892192 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # generate new key for this host
 # Copyright (C) 2001, 2002  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_plutoload/_plutoload.in openswan-2.2.0/programs/_plutoload/_plutoload.in
--- openswan-2.2.0.orig/programs/_plutoload/_plutoload.in	2002-09-16 00:45:14.000000000 +0200
+++ openswan-2.2.0/programs/_plutoload/_plutoload.in	2004-11-24 20:34:07.092479784 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # Pluto database-loading script
 # Copyright (C) 1998, 1999, 2001  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_plutorun/_plutorun.in openswan-2.2.0/programs/_plutorun/_plutorun.in
--- openswan-2.2.0.orig/programs/_plutorun/_plutorun.in	2004-06-09 04:58:30.000000000 +0200
+++ openswan-2.2.0/programs/_plutorun/_plutorun.in	2004-11-24 20:34:14.798308320 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # Pluto control daemon
 # Copyright (C) 1998, 1999, 2001  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_realsetup/_realsetup.in openswan-2.2.0/programs/_realsetup/_realsetup.in
--- openswan-2.2.0.orig/programs/_realsetup/_realsetup.in	2004-06-09 05:00:41.000000000 +0200
+++ openswan-2.2.0/programs/_realsetup/_realsetup.in	2004-11-24 20:34:21.625270464 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # IPsec startup and shutdown command
 # Copyright (C) 1998, 1999, 2001  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_secretcensor/_secretcensor.in openswan-2.2.0/programs/_secretcensor/_secretcensor.in
--- openswan-2.2.0.orig/programs/_secretcensor/_secretcensor.in	2002-04-24 09:36:06.000000000 +0200
+++ openswan-2.2.0/programs/_secretcensor/_secretcensor.in	2004-11-24 20:34:30.398936664 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # implements secret censoring for barf
 # Copyright (C) 1999  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/secrets/secrets.in openswan-2.2.0/programs/secrets/secrets.in
--- openswan-2.2.0.orig/programs/secrets/secrets.in	2003-10-31 03:39:44.000000000 +0100
+++ openswan-2.2.0/programs/secrets/secrets.in	2004-11-24 20:37:31.732369792 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # program which prompts for PINs and passphrases
 # alias for ipsec auto --rereadsecrets
 # Copyright (C) 2003  Andreas Steffen
diff -uNr openswan-2.2.0.orig/programs/send-pr/send-pr.in openswan-2.2.0/programs/send-pr/send-pr.in
--- openswan-2.2.0.orig/programs/send-pr/send-pr.in	2003-07-14 14:26:17.000000000 +0200
+++ openswan-2.2.0/programs/send-pr/send-pr.in	2004-11-24 20:37:59.095210008 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # Submit a problem report to a GNATS site.
 # Copyright (C) 2001 Milan Zamazal
 # Copyright (C) 1993, 2001 Free Software Foundation, Inc.
diff -uNr openswan-2.2.0.orig/programs/setup/setup.in openswan-2.2.0/programs/setup/setup.in
--- openswan-2.2.0.orig/programs/setup/setup.in	2004-07-16 14:57:08.000000000 +0200
+++ openswan-2.2.0/programs/setup/setup.in	2004-11-24 20:38:08.137835320 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # IPsec startup and shutdown script
 # Copyright (C) 1998, 1999, 2001  Henry Spencer.
 # Copyright (C) 2002              Michael Richardson <mcr@freeswan.org>
diff -uNr openswan-2.2.0.orig/programs/showdefaults/showdefaults.in openswan-2.2.0/programs/showdefaults/showdefaults.in
--- openswan-2.2.0.orig/programs/showdefaults/showdefaults.in	2002-05-31 03:21:53.000000000 +0200
+++ openswan-2.2.0/programs/showdefaults/showdefaults.in	2004-11-24 20:38:27.257928624 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # show defaults for %defaultroute
 # Copyright (C) 2000  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/showhostkey/showhostkey.in openswan-2.2.0/programs/showhostkey/showhostkey.in
--- openswan-2.2.0.orig/programs/showhostkey/showhostkey.in	2004-03-24 22:08:22.000000000 +0100
+++ openswan-2.2.0/programs/showhostkey/showhostkey.in	2004-11-24 20:36:51.905424408 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 # show key for this host, in DNS (or other) format
 # Copyright (C) 2000, 2001  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_startklips/_startklips.in openswan-2.2.0/programs/_startklips/_startklips.in
--- openswan-2.2.0.orig/programs/_startklips/_startklips.in	2004-07-15 04:34:42.000000000 +0200
+++ openswan-2.2.0/programs/_startklips/_startklips.in	2004-11-24 20:34:37.087919784 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # KLIPS startup script
 # Copyright (C) 1998, 1999, 2001, 2002  Henry Spencer.
 # 
diff -uNr openswan-2.2.0.orig/programs/_updown_x509/_updown_x509.in openswan-2.2.0/programs/_updown_x509/_updown_x509.in
--- openswan-2.2.0.orig/programs/_updown_x509/_updown_x509.in	2003-10-31 03:31:13.000000000 +0100
+++ openswan-2.2.0/programs/_updown_x509/_updown_x509.in	2004-11-24 20:35:06.560439280 +0100
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 #
 # customized updown script
 #
