diff -NurpP --minimal openswan-2.4.7_clean/Makefile.inc openswan-2.4.7/Makefile.inc
--- openswan-2.4.7_clean/Makefile.inc	2007-04-24 10:08:25.000000000 +0200
+++ openswan-2.4.7/Makefile.inc	2007-04-24 10:09:47.000000000 +0200
@@ -334,7 +334,7 @@ endif
 # should we include all manner of known to be broken/weak?
 # use this only if you are building some kind of a testing
 # device. Normal use does not need any of this.
-USE_WEAKSTUFF?=false
+USE_WEAKSTUFF?=true
 
 # should we include additional algorithms? It adds lots of
 # code space increases risk of bad crypto code, but offers possibilities 
diff -NurpP --minimal openswan-2.4.7_clean/programs/pluto/Makefile openswan-2.4.7/programs/pluto/Makefile
--- openswan-2.4.7_clean/programs/pluto/Makefile	2007-04-24 10:08:25.000000000 +0200
+++ openswan-2.4.7/programs/pluto/Makefile	2007-04-24 10:11:50.000000000 +0200
@@ -210,7 +210,7 @@ CURL_LIBS=-lcurl
 endif
 
 ifeq ($(USE_WEAKSTUFF),true)
-WEAK_DEFS=-DUSE_VERYWEAK_DH1=1 #-DUSE_1DES
+WEAK_DEFS=-DUSE_VERYWEAK_DH1=1 -DUSE_1DES
 endif
 
 ifeq ($(USE_EXTRACRYPTO),true)
