--- openswan-2.6.23/programs/showpolicy/showpolicy.c~	2009-09-09 02:42:54.000000000 +0200
+++ openswan-2.6.23/programs/showpolicy/showpolicy.c	2010-05-04 13:56:15.000000000 +0200
@@ -122,7 +122,7 @@
 	int packetcount =0;
 	
 	do {
-		unsigned int pktref[1];
+		unsigned int pktref[2] = {0, 0};
 
 		memset(&from, 0, sizeof(from));
 		memset(&to,   0, sizeof(to));
--- openswan-2.6.25/include/ipsecconf/keywords.h~	2010-03-21 18:39:35.000000000 +0100
+++ openswan-2.6.25/include/ipsecconf/keywords.h	2010-05-04 14:11:46.000000000 +0200
@@ -160,8 +160,8 @@
     KNCF_MAX              
 };
 
-#define KEY_STRINGS_MAX (KSF_MAX > KSCF_MAX ? KSF_MAX : KSCF_MAX)+1
-#define KEY_NUMERIC_MAX (KBF_MAX > KNCF_MAX ? KBF_MAX : KNCF_MAX)+1
+#define KEY_STRINGS_MAX ((int)KSF_MAX > (int)KSCF_MAX ? (int)KSF_MAX : (int)KSCF_MAX)+1
+#define KEY_NUMERIC_MAX ((int)KBF_MAX > (int)KNCF_MAX ? (int)KBF_MAX : (int)KNCF_MAX)+1
 
 /* these are bits set in a word */
 enum keyword_valid {
