diff -burNp openswan-2.6.23/Makefile.inc openswan-2.6.23-dud/Makefile.inc
--- openswan-2.6.23/Makefile.inc	2009-09-09 02:42:54.000000000 +0200
+++ openswan-2.6.23-dud/Makefile.inc	2009-11-05 21:28:11.785688740 +0100
@@ -49,7 +49,7 @@ SHELL=/bin/sh
 DESTDIR?=
 
 # "local" part of tree, used in building other pathnames
-INC_USRLOCAL=/usr/local
+INC_USRLOCAL=/usr
 
 # PUBDIR is where the "ipsec" command goes; beware, many things define PATH
 # settings which are assumed to include it (or at least, to include *some*
@@ -58,7 +58,7 @@ PUBDIR=$(DESTDIR)$(INC_USRLOCAL)/sbin
 
 # BINDIR is where sub-commands get put, FINALBINDIR is where the "ipsec"
 # command will look for them when it is run. Also called LIBEXECDIR.
-FINALLIBEXECDIR?=$(INC_USRLOCAL)/libexec/ipsec
+FINALLIBEXECDIR?=$(INC_USRLOCAL)/lib/ipsec
 LIBEXECDIR=$(DESTDIR)$(FINALBINDIR)
 
 FINALBINDIR=${FINALLIBEXECDIR}
@@ -78,7 +78,7 @@ LIBDIR=$(DESTDIR)$(FINALLIBDIR)
 # location within INC_USRLOCAL
 INC_MANDIR=man
 # the full pathname
-MANTREE=$(DESTDIR)$(INC_USRLOCAL)/$(INC_MANDIR)
+MANTREE=$(DESTDIR)$(INC_USRLOCAL)/share/$(INC_MANDIR)
 # all relevant subdirectories of MANTREE
 MANPLACES=man3 man5 man8
 
--- openswan-2.6.31/programs/examples/Makefile.orig	2010-10-18 19:13:36.000000000 +0300
+++ openswan-2.6.31/programs/examples/Makefile	2010-10-23 13:57:30.737483665 +0300
@@ -15,7 +15,7 @@
 OPENSWANSRCDIR?=$(shell cd ../..; pwd)
 include ${OPENSWANSRCDIR}/Makefile.inc
 
-CONFDSUBDIR=examples
+CONFDSUBDIR=.
 CONFDFILES=l2tp-cert.conf l2tp-psk.conf linux-linux.conf sysctl.conf xauth.conf ipv6.conf hub-spoke.conf oe-exclude-dns.conf mast-l2tp-psk.conf
 
 include ${srcdir}../Makefile.program
